ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -3027,7 +3025,9 @@ periodics:
       - ci-operator
       env:
       - name: RELEASE_IMAGE_LATEST
-        value: registry.svc.ci.openshift.org/ocp/release:4.2
+        value: registry.svc.ci.openshift.org/ocp/release:4.2-ci
```

Comment:
If you call it `ocp` in the name then this has to be `4.2` and it means you're pulling nightlies.

If you call it `origin` in the name then this can be `4.2-ci` and you're pulling CI payloads

Reply:
Updated.